src/App.js

```diff
@@ -7,51 +7,64 @@ class App extends Component {
   render() {
     return (
       <section className="container-fluid image-lukasz">
-        <div className="row">
+        <div className="row ">
           <div className="container">
-            <div className="row">
+            <div className="row justify-content-end">
               <header>
-                <div className="col" />
-                <nav>
-                  <ul>
-                    <li>Umiejętności</li>
-                    <li>Kontakt</li>
-                  </ul>
-                </nav>
+                <div className="col ">
+                  <nav className="nav-bar ">
+                    <ul className="nav ">
+                      <li className="nav-item">
+                        <a className="nav-link font-weight-bold" href="#">
+                          Umiejętności
+                        </a>
+                      </li>
+                      <li>
+                        <a className="nav-link font-weight-bold" href="#">
+                          Kontakt
+                        </a>
+                      </li>
+                    </ul>
+                  </nav>
+                </div>
               </header>
             </div>
-            <div className="row">
+            <div className="row d-flex align-items-end">
               <div className="left-box col-4">
-                <div className="title-box justify-content-center">
-                  <h5 className="title ">Jestę frontendowcę</h5>
+                <div className="title-box justify-content-center sticky-top">
+                  <h6 className="title">Jestę frontendowcę</h6>
                 </div>
                 <div className="inside-text">
-                  <h3>
-                    <strong>Cześć!</strong>
-                  </h3>
-                  <p>
-                    Jestem web developerem i rozwijam się na ścieżce javascriptu
+                  <h3>Cześć!</h3>
+                  <p className="">
+                    Jestem web developerem i&nbsp;rozwijam się na ścieżce
```

Comment:
to tylko pytanie, czy potrzebna jest to twarde złamanie lini? nie lepiej aby się łamało według wolnego miejsca?

Reply:
Potrzebne jest bo według formatownia tektu nie wolno zostawiac "i" "z" "o" itp. na końcu lini

Reply:
wydaje mi się, że popsuje CI to widok na rwd. Ale to wyjdzie podczas testowania :) przechodzi narazie dalej.